### DISCUSSION TITLE
Unique Drop Rates?

### DISCUSSION THREAD

**Discussion by Chonkblonk** [diasurgical/DevilutionX#6998] [Q&A] [answered]
How well do you guys get treated with unique drops?

Very reliably when a unique ring gets dropped, it's virtually always Gladiator's Ring, I have like 7 or 8 of them now (I sold one out of frustration somewhere along the way, but it's starting to get almost funny now so I save them).
Just the same, I seem to have gotten Shirotachi to drop a number of times, and the second time was just 15 minutes later in the same game and on the same seed. Stormshield, Blackoak Shield, Wicked Axe, and Needler, are all uniques which will drop very often.
Meanwhile, Lazarus has yet to drop Naj's Light Plate (but I did eventually get my precious Dreamflange, along with a spare one later).

I seem to remember that Hellfire affects how uniques are dropped, but DevilutionX I think at least fixes it partially, doesn't it? I have a memory of people complaining that uniques are virtually impossible to get at all in original unpatched Hellfire. By reading really old materials like Jarulf's Guide, I understand that there's typically some kind of 'sequencing' where some uniques need to drop before others will be able to, but that doesn't seem to be quite the case here.

I notice also that some seeds appear to be more generous with dropping oils too, either just blacksmith oils, or they drop the other ones, in which case I find it compelling to at least farm the Cathedral after Lazarus has once again given me nothing of value.

**Comment by DakkJaniels** [comment]
It works like Jarulf's Guide describes, no changes.

Edit: I guess it's not exactly as JG describes, at least for Hellfire, because he only states that the highest qlvl item will drop, but it's actually dependent on the order the unique items are listed in the array. Before Hellfire, they were in ascending level order, but since Hellfire added the unique rings to the end of the array, with Gladiator's Ring being the last one, it is the one that will always drop in multiplayer if it is in the list of possible drops from the source (monster, barrel, chest, etc.).

**Comment by Chonkblonk** [comment]
Right, so is there some method which would be helpful for hoping to score Naj's Light Plate?
Is there some other armor I should expect to drop before I can get that one?
And do you other guys also get LOTS of Gladiator's Rings?

**Reply by AJenbo** [reply]
you need to get them during the same session or it will reset

**Reply by DakkJaniels** [reply, marked as answer]
> is there some method which would be helpful for hoping to score Naj's Light Plate?

Be luckier? 

> Is there some other armor I should expect to drop before I can get that one?

No, it is the only plate mail, per JG, so you are just unlucky. JG says about 30 games in Hellfire and 40 games in Diablo

> And do you other guys also get LOTS of Gladiator's Rings?

If you are playing Hellfire Multiplayer, and you are only playing where monster level (for drops, i.e., qlvl) is 10 or higher, you are only going to get Gladiator's ring.

Edit: I guess it's not exactly as JG describes, because he only states that the highest qlvl item will drop, but it's actually dependent on the order the unique items are listed in the array. Before Hellfire, they were ascending level order, but since Hellfire added the unique rings to the end of the array, with Gladiator's Ring being the last one, it is the one that will always drop in multiplayer if it is in the list of possible drops from the source (monster, barrel, chest, etc.).

**Reply by Chonkblonk** [reply]
> you need to get them during the same session or it will reset

That I know.

> Be luckier?

Will attempt this.

**Reply by Chonkblonk** [reply]
It worked! Didn't get it from Lazarus though, I got it from Bloodlust.
![bild](https://github.com/diasurgical/devilutionX/assets/114463804/cc4ebc46-5d64-4dbc-8d85-7f40e9370676)

The funny thing is also that I figured I had gotten a good idea of how drops functioned. I noticed that when I would get books dropping from enemies, I could often expect Lazarus to drop one (and sometimes a hilariously worthless one like Warp), thus I decided to test it out this run, went to kill Leoric and he gave me one, and I figured I would probably get one from Lazarus too, but hey, may as well test and see if my observation holds up, and then BAM, my grail armor!

**Reply by AJenbo** [reply]
grats
